app/views/contact_manager/_contact_field.haml

```diff
@@ -1,4 +1,7 @@
-- label = object.label_for(contact_type)
+- match_label ||= nil
```

Comment:
That's an odd syntax since it will only call the = side of the equation if match_label is nil.  Maybe you meant `defined?`?